include/experimental/__p0009_bits/extents.hpp

```diff
@@ -242,8 +263,16 @@ class extents
   MDSPAN_TEMPLATE_REQUIRES(
     class SizeType, size_t N,
     /* requires */ (
-      (N == rank() || N == rank_dynamic()) &&
       _MDSPAN_TRAIT(is_convertible, SizeType, size_type)
+// TODO: check whether this works with newest NVCC, doesn't with 11.4
```

Comment:
Does `rank()` and `rank_dynamic()` not work at all with NVCC or just in the REQUIRES or FOLD_AND macros?

Reply:
in most cases its correct, just in that fold expression as part of some funky enable ifing it falls apart.